tests/Hooks/AfterTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+test('after helper can be chained', function () {
+    expect(true)->toBeTrue();
+})->after(function () {
+    // Example
+});
+
+test('after closure is called', function () use (&$afterWasCalled) {
+    $afterWasCalled = true;
+
+    expect(true)->toBeTrue();
+})->after(function () use (&$afterWasCalled) {
+    expect($afterWasCalled)->toBeTrue();
```

Comment:
Commenting out the following line should fail the test:
```php
($this->after)();
```

It seems that this test pass even when a callback is never executed. If that's the case, the test is not really testing anything. The test should fail when the functionality under test is broken (the callback is never called).